src/components/PortForwarding.vue

```diff
@@ -81,10 +112,18 @@ export default {
       }));
     },
   },
+  mounted() {
+    ipcRenderer.on('k8s-check-state', (event, stt) => {
+      this.$data.k8sState = stt;
```

Comment:
What does `stt` stand for? Why not call it `state` or `newState` or something?

Reply:
We should call this `Jonah coding`, related to cargo-cult programming. Jonah's the son of a friend of ours, whose grade-7 yearbook caption was "It was like that when I got here".